app/helpers/metadata_form_helper.rb

```diff
@@ -1,10 +1,19 @@
 # frozen_string_literal: false
 module MetadataFormHelper
   def form_title(params)
-    if form_custom_title(params)
-      "#{params['action'].capitalize} #{params['change_set'].humanize.singularize.downcase} resource"
+    controller_name = params["controller"]
+    model_name = controller_name.singularize.downcase
+    action = params["action"].capitalize
+    action = "New" if action == "Create"
+
+    model_params = params[model_name]
+
+    if model_params && form_custom_title(model_params)
+      "#{action} #{model_params['change_set'].humanize.singularize.downcase} resource"
```

Comment:
Without this, when `create` actions raise errors, the title for the form will just be `Collection` (even when the `change_set` is set properly to `archival_media_collection`)